src/context/userDataContext/watchLater-serverCalls.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+const getWatchlater = async (dataDispatch) => {
+  const token = localStorage.getItem("userToken");
+  const {
+    data: { watchlater },
+  } = await axios.get("/api/user/watchlater", {
+    headers: { authorization: token },
+  });
+  dataDispatch({ type: "WATCHLATER", payload: watchlater });
+};
```

Comment:
consider wrapping api calls in a try catch block

Reply:
Ooh thanks, I think I missed it here!
I will correct it in the next branch 🤯